### ISSUE TITLE
News API

### ISSUE THREAD

**Issue by WebFreak001** [ToontownRewritten/api-doc#8]
So by trying some random URLs I found out about https://www.toontownrewritten.com/api/news but I don't quite know yet how to list multiple news. Is there a way for this or should I just GET the news endpoint and then subtract the id a few times by one and then always GET https://www.toontownrewritten.com/api/news/:id

I think it would be easier on the servers if there was an endpoint directly giving the latest 10 news for example instead of making 10 requests

**Comment by jjkoletar** [comment]
Yeah there is a length parameter that you can pass to `/api/news/list`. I need to get around to documenting this... 🐬
